src/utils/session-helper.js

```diff
@@ -290,9 +290,11 @@ const createFileChecksum = async (fileObject) => {
 /**
  * Helper Function that returns Driver's License ttl file based off of image passed
  *
+ * @function createDriversLicenseTtlFile
  * @memberof utils
  * @param {fileObjectType} fileObject - Object containing information about file
  * @param {string} documentUrl - url of uploaded document or resource
+ * @param {object} checksum - checksum for uploaded file
  */
```

Comment:
I believe this function returns a `ThingLocal` type. VSCode is nice given the fact it integrates TypeScript typings even if you missed the `@returns` from JSDoc.

<img width="778" alt="Screen Shot 2023-05-24 at 4 33 40" src="https://github.com/codeforpdx/PASS/assets/14917816/84a35008-2c08-4fc4-aa6b-4f0d378cece9">

But it's better to have `@returns` if you wish to have a more complete documentation of the function.